tests/commands/execution/test_ssh.py

```diff
@@ -0,0 +1,68 @@
+import pathlib
+import subprocess
+import time
+
+from tests.commands.execution.test_watch import no_sleep
+from tests.commands.execution.utils import get_execution_data_mock
+from tests.fixture_data import EXECUTION_DATA
+from valohai_cli.commands.execution.ssh import ssh
+
+
+def test_ssh_in_completed_execution(runner, logged_in_and_linked, monkeypatch):
+    with get_execution_data_mock():
+        counter = EXECUTION_DATA["counter"]
+        output = runner.invoke(ssh, [str(counter)], catch_exceptions=False).output
+        assert f"Error: Execution #{counter} is complete. Cannot SSH into it.\n" in output
+
+        EXECUTION_DATA["status"] = "queued"
+        monkeypatch.setattr(time, "sleep", no_sleep)
+
+
+def test_ssh_in_queued_execution(runner, logged_in_and_linked, monkeypatch):
+    counter = EXECUTION_DATA["counter"]
+    EXECUTION_DATA["status"] = "queued"
+    monkeypatch.setattr(time, "sleep", no_sleep)
+    with get_execution_data_mock():
+        output = runner.invoke(ssh, [str(counter)], catch_exceptions=False).output
+        assert f"Execution #{counter} is queued. Waiting for it to start...\n" in output
+
+
+def test_ssh_with_no_ssh_details_present(runner, logged_in_and_linked, monkeypatch):
+    counter = EXECUTION_DATA["counter"]
+    EXECUTION_DATA["status"] = "started"
+    monkeypatch.setattr(time, "sleep", lambda x: None)
+    monkeypatch.setattr("valohai_cli.utils.ssh.find_ssh_details", lambda x: None)
+    with get_execution_data_mock():
+        output = runner.invoke(ssh, [str(counter)], catch_exceptions=False).output
+        assert "1/5 Retrying: No SSH details found...\n" in output
+        assert "2/5 Retrying: No SSH details found...\n" in output
+        assert "3/5 Retrying: No SSH details found...\n" in output
+        assert "4/5 Retrying: No SSH details found...\n" in output
+        assert "5/5 Retrying: No SSH details found...\n" in output
+
+
+def test_ssh(runner, logged_in_and_linked, monkeypatch):
+    counter = EXECUTION_DATA["counter"]
+    EXECUTION_DATA["status"] = "started"
+    monkeypatch.setattr(time, "sleep", lambda x: None)
+
+    def mock_prompt():
+        return pathlib.Path("/path/to/private")
```

Comment:
Add the `tmp_path` fixture to the test and return that instead of hard-coding a path:
```suggestion
    def mock_prompt():
        return tmp_path
```